### PR TITLE
Making Verses Clickable. Added hover mouse option

### PR DESCRIPTION
**Pull request by Salomigandra**

### CODE REVIEW

**Review by brandonxia01 — APPROVED** [review]
Looks good to me!

File: biblego/components/screens/ChapterViewer.tsx [hunk 1]
```diff
@@ -77,11 +77,12 @@ const BibleVerseDisplay: React.FC<BibleVerseDisplayProps> = ({
             <span
               onClick={(e) => handleVerseSelection(bibleVerse.verseNumber, e)}
               // Make verse number bolder if it has a paragraph break, like nasb
-              className={`relative inline-flex items-center justify-center w-[15px] h-[19px] mr-2 bg-purple-200 rounded-[1px] text-center text-gray-600 text-md font-source-serif leading-[23px] font-semibold select-none`}>
+              className={`relative inline-flex items-center justify-center w-[15px] h-[19px] mr-2 bg-purple-200 rounded-[1px] text-center text-gray-600 text-md font-source-serif leading-[23px] font-semibold select-none cursor-pointer hover:bg-purple-200`}>
               {bibleVerse.verseNumber}
             </span>
             <span
-              className={`font-source-serif font-normal leading-[23px] select-none ${
+              className={`font-source-serif font-normal leading-[23px] cursor-pointer
```
Comment (brandonxia01) [inline biblego/components/screens/ChapterViewer.tsx:84]: @Salomigandra it looks like your merge from main lost the `select-none` you added before.

Just pull from main, make sure there's the changes you had from both PRs, and put up another pr.

Comment (Salomigandra) [reply biblego/components/screens/ChapterViewer.tsx:84]: Okay 👍
